apps/balancer-tools/src/app/apr/rounds/[roundId]/route.ts

```diff
@@ -0,0 +1,15 @@
+import { NextRequest, NextResponse } from "next/server";
+
+import { DuneAPI } from "#/lib/dune";
+import { voteGaugeByID } from "../../mock_apis";
+
+export async function GET(
```

Comment:
Not sure about this, did this way since we dont want to use DuneAPI on the client

Reply:
Got it, that makes sense! 

I think that's the default way to reason about this -- but we could could also leverage server components like I did in my PR to load it (I especially enjoy it because I can forget about local loading states, just leaving `Suspense` handle that 😅 )